src/service/authentication-service.ts

```diff
@@ -137,18 +127,19 @@ export default class AuthenticationService {
    * @param token - The JWT token that can be used to authenticate.
    * @returns The authentication response.
    */
-  public static asAuthenticationResponse(
+  public static async asAuthenticationResponse(
```

Comment:
Why is this async?

Reply:
Leftover from an earlier (uglier) implementation, fixed now.